.github/workflows/release.yml

```diff
@@ -0,0 +1,196 @@
+name: Release
+on:
+  release:
+    types:
+      - created
+      # Temporary, for testing
+      - edited
+      - published
+
+# Test trigger. Uncomment to test basic flow.
```

Comment:
Leftover?

Reply:
No, Kept as a prompt for the next unlucky person who needs to modify this flow, to get the basics right without having to disturb other people with test releases